### PR TITLE
add sigmut

### PR DESCRIPTION
**Pull request by lgpdv**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by CJ-Wright** [comment]
How is this one different from https://github.com/conda-forge/staged-recipes/pull/10623

**Comment by lgpdv** [comment]
> How is this one different from #10623
It's the same PR, some minor changes. I've commit from a different user, since I've to move as well the repo.
Regarding the possibility of installing with pip, actually is a modified version of the original tool and I can not install now from the wrapper script.

File: recipes/sigmut/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - matplotlib >=3.1.0
```
Comment (scopatz) [inline recipes/sigmut/meta.yaml]: These should probably be matplotlib-base

File: recipes/sigmut/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{% set name = "sigmut" %}
+{% set version = "v1.0" %}
```
Comment (scopatz) [inline recipes/sigmut/meta.yaml]: Our version numbers don't contain `v`

File: recipes/sigmut/meta.yaml (hunk 1 above)
Comment (lgpdv) [reply recipes/sigmut/meta.yaml]: Thanks a lot for your suggestion. I will modified the file and try the PR again.

File: recipes/sigmut/meta.yaml (hunk 2 above)
Comment (lgpdv) [reply recipes/sigmut/meta.yaml]: I've seen some other recipes using 'v' for versions. Anyhow, thanks again for your suggestion; I will modified the file and try the PR again.

File: recipes/sigmut/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  url: https://github.com/lgpdv/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
```
Comment (CJ-Wright) [inline recipes/sigmut/meta.yaml]: Are all of these host requirements? Usually host requirements for noarch python builds are only python and pip.

Comment (lgpdv) [reply recipes/sigmut/meta.yaml]: Yes they are. What is weird is that locally I can create the conda repository and upload without any problem to my channel (conda install -c lgpdv sigmut).

**Comment by lgpdv** [comment]
Hello @CJ-Wright and @scopatz . After some modifications the tool has succeeded almost all checks, except one. Do you think you can help me to solve it? Thank you very much.

File: recipes/sigmut/meta.yaml (hunk 3 above)
Comment (lgpdv) [reply recipes/sigmut/meta.yaml]: Hello @CJ-Wright . Finally all tests have succeeded. I think the PR is ready to be merged now.

File: recipes/sigmut/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/lgpdv/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: generic
+  number: 0
+  skip: True
```
Comment (mariusvniekerk) [inline recipes/sigmut/meta.yaml]: This skips all builds.  Are you sure?

Comment (lgpdv) [reply recipes/sigmut/meta.yaml]: Thank you for this notice @mariusvniekerk . I've edited the PR but some problems arose related to the linting of the recipe.

**Pull request closed without merging**